Src/Support/IntegrationTests_NetCore/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace IntegrationTests_NetCore
+{
+    public class Program
+    {
+        public static void Main(string[] args)
```

Comment:
Ditto. (Are these just skeletons to make it easy to write an ad-hoc test? If so, that makes sense - a comment would be useful though.)